tests/unit/Handlers/PagesHandlerTest.php

```diff
@@ -79,6 +79,65 @@ public function testGetPageShouldReturnStatus404(): void
         $this->assertEquals(404, $response->getStatusCode());
     }
 
+    public function testFindPagesShouldReturnStatus200(): void
+    {
+        $request = Request::createFromEnvironment(
+            Environment::mock([
+                'REQUEST_METHOD' => 'POST',
+                'REQUEST_URI' => '/api/pages'
```

Comment:
Should these uris be `/api/pages/search`?

Reply:
Yep